src/tts_text/phoneme_covering.py

```diff
@@ -0,0 +1,194 @@
+"""Module for creating a phoneme covering set."""
+
+import json
+import re
+import logging
+from collections import Counter
+from pathlib import Path
+from tqdm.auto import tqdm
+
+from datasets import load_dataset, Dataset
+from omegaconf import DictConfig
+
+logger = logging.getLogger(__name__)
+
+
+def sort_wiki_by_phoneme_occurence(cfg: DictConfig) -> Dataset:
+    """Sort the wiki dataset by phoneme occurence.
+
+    Args:
+        cfg: The Hydra configuration object.
+
+    Returns:
+        The sorted dataset.
+
+    Raises:
+        ValueError: If sort_by is not one of "da", "en" or "all".
+    """
+    # The `wiki40b` dataset is a small dataset so we can load it all into memory
+    # instead of streaming it.
+    dataset = load_dataset(
+        "wiki40b",
+        "da",
+        split="train",
+        beam_runner="DirectRunner",
+    )
+
+    with Path(cfg.dirs.phoneme_dir).open() as f:
+        phonemes = json.load(f)
```

Comment:
If we change the phonemes.json file to be part of the config, then this block should change.